python/lsst/ts/fbs/utils/maintel/basis_functions.py

```diff
@@ -207,6 +208,71 @@ def get_basis_functions_ddf_survey(
     ]
 
 
+def get_basis_functions_field_survey(
+    nside: int,
+    wind_speed_maximum: float = 10.0,
```

Comment:
This is really low. We don't have a good sense of our wind speed limits, but I think you want to push this up to 20.0 m/s to avoid targets getting masked too often.

Reply:
On second thought, I wonder if we should just remove the avoidWind basis function all together (or make a boolean option to toggle it on/off). We might learn something valuable early on about the effects of wind shake by leaving it out that would likewise apply to the main camera.

Reply:
On second thought, I wonder if we should just remove the avoidWind basis function all together (or make a boolean option to toggle it on/off). We might learn something valuable early on about the effects of wind shake by leaving it out that would likewise apply to the main camera.

Reply:
Setting to 20.0 m/s for now. Could set to a higher value.